src/components/StatusChip/StatusChip.styles.tsx

```diff
@@ -0,0 +1,33 @@
+import { Chip, styled } from '@mui/material';
+
+import { StatusChipProps } from './StatusChip.types';
+
+const STATUS = {
+    COMPLETED: 'Completed',
+    CANCELLED: 'Cancelled',
+    IN_PROGRESS: 'In progress',
+};
+
+export const StyledStatusChip = styled(Chip, {
+    shouldForwardProp: (prop) => prop !== 'status',
+})<StatusChipProps>(({ theme, status }) => {
+    switch (status) {
+        case STATUS.COMPLETED:
+            return {
+                color: theme.palette.primary.dark,
+                backgroundColor: theme.palette.primary.light,
+            };
+        case STATUS.IN_PROGRESS:
+            return {
+                color: theme.palette.secondary.dark,
+                backgroundColor: theme.palette.secondary.light,
+            };
+        case STATUS.CANCELLED:
+            return {
+                color: theme.palette.error.dark,
+                backgroundColor: theme.palette.error.light,
+            };
+        default:
+            return {};
+    }
+});
```

Comment:
What suggesting to implement something like this
```suggestion
export const StyledStatusChip = styled(Chip, {
    shouldForwardProp: (prop) => prop !== 'status',
})<StatusChipProps>(({ theme, status }) => {
const STATUS_BG_COLOR_MAP = {
    'Completed': theme.palette.primary.light,
    'Cancelled': theme.palette.secondary.light,
    'In progress': theme.palette.error.light,
};
            return {
                color: theme.palette.primary.dark,
                backgroundColor: STATUS_BG_COLOR_MAP[status],
            };
});
```